homeassistant/components/media_player/samsungtv.py

```diff
@@ -247,6 +250,22 @@ def media_previous_track(self):
         """Send the previous track command."""
         self.send_key('KEY_REWIND')
 
+    def play_media(self, media_type, media_id, **kwargs):
+        """Support changing a channel."""
+        if media_type != MEDIA_TYPE_CHANNEL:
+            raise NotImplementedError()
+
+        # media_id should only be a channel number
+        try:
+            if int(media_id) < 1:
```

Comment:
Use our voluptuous validator `positive_int` in config_validation helper module.

Reply:
thank you for the comments @MartinHjelmare !! I'm not super comfortable in python/voluptuous but from what I've read seems like `positive_int` would just be a callback that would throw an exception, right? so I should just call `cv. positive_int(media_id)` (after importing) and it should either just explode or be fine and move along?

Reply:
Yes, but you can `except vol.Invalid` to catch the invalid value error and take appropriate action.